README.md

```diff
@@ -172,15 +168,15 @@ Learn more about <a href="https://www.builder.io/c/docs/how-builder-works-techni
 
 ## How is the performance?
 
-Performance is our top priority. All content is delivered from the edge, renderable server side or statically, we [optimize the heck out of it](https://www.builder.io/blog/high-performance-no-code) and our SDKs are small and mighty. 
+Performance is our top priority. All content is delivered from the edge, renderable server side or statically, we [optimize the heck out of it](https://www.builder.io/blog/high-performance-no-code) and our SDKs are small and mighty.
 
-Some example SDK sizes from Bundlephobia:
+Some example SDK sizes from Bundle.js:
 
-|SDK|Size|
-|----|----|
-|React|![React Bundle Size](https://img.shields.io/bundlephobia/minzip/@builder.io/sdk-react)|
-|Vue|![Vue Bundle Size](https://img.shields.io/bundlephobia/minzip/@builder.io/sdk-vue)|
-|Qwik|![Vue Bundle Size](https://img.shields.io/bundlephobia/minzip/@builder.io/sdk-qwik)|
+| SDK   | Size                                                                                                                                                                                                                             |
+| ----- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| React | ![React Bundle Size](https://deno.bundlejs.com/badge?q=@builder.io/sdk-react&treeshake=[*]&badge=detailed)                                                                                                                       |
+| Vue   | ![Vue Bundle Size](https://deno.bundlejs.com/badge?q=@builder.io/sdk-vue/vue3&treeshake=[*]&badge=detailed&config={%22esbuild%22:{%22external%22:[%22@vue/runtime-dom%22,%22@vue/runtime-core%22,%22@vue/shared%22,%22vue%22]}}) |
+| Qwik  | ![Qwik Bundle Size](https://deno.bundlejs.com/badge?q=@builder.io/sdk-qwik&treeshake=[*]&config={"esbuild":{"external":["@builder.io/qwik"]}}&badge=detailed)                                                                    |
```

Comment:
are you sure you want `badge=details`? it's a lot larger and has some not useful info (like ungzipped sizes)

Reply:
also these estimates are kinda huge, are they accorate?

![](https://deno.bundlejs.com/badge?q=@builder.io/sdk-vue/vue3&treeshake=[*]&badge=detailed&config={%22esbuild%22:{%22external%22:[%22@vue/runtime-dom%22,%22@vue/runtime-core%22,%22@vue/shared%22,%22vue%22]}})

![](https://deno.bundlejs.com/badge?q=@builder.io/sdk-qwik&treeshake=[*]&config={"esbuild":{"external":["@builder.io/qwik"]}}&badge=detailed)

90kb is way larger than the old sdk which doesn't make a lot of sense

Reply:
Yeah...because we added https://github.com/NeilFraser/JS-Interpreter which is massive.

Reply:
the `sdk-react` size was low because of a bug. I just re-released it and it's also in line with those numbers:

![](https://deno.bundlejs.com/badge?q=@builder.io/sdk-react&treeshake=[*])